apps/lfx-one/src/app/modules/dashboards/health-metrics/health-metrics-full-page-empty-state/health-metrics-full-page-empty-state.component.html

```
@@ -0,0 +1,80 @@
+<!-- Copyright The Linux Foundation and each contributor to LFX. -->
+<!-- SPDX-License-Identifier: MIT -->
+
+<div class="flex flex-col items-center gap-8 py-16 text-center" data-testid="health-metrics-full-page-empty-state">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)